www/src/graph/payments.graphql

```diff
@@ -114,36 +136,88 @@ query Subscription {
 }
 
 mutation UpdateAccountBilling($attributes: AccountAttributes!) {
-  updateAccount(attributes: $attributes) { id }
+  updateAccount(attributes: $attributes) {
+    id
+  }
 }
 
-mutation UpgradeToProfessionalPlan($planId: ID!, $billingAddress: AddressAttributes) {
+mutation UpgradeToProfessionalPlan(
+  $planId: ID!
+  $billingAddress: AddressAttributes
+) {
   createPlatformSubscription(planId: $planId, billingAddress: $billingAddress) {
     id
     latestInvoice {
       ...Invoice
-      paymentIntent { ...PaymentIntent }
+      paymentIntent {
+        ...PaymentIntent
+      }
     }
   }
 }
 
 mutation DowngradeToFreePlanMutation {
-  deletePlatformSubscription { id }
+  deletePlatformSubscription {
+    id
+  }
 }
 
 query Cards {
   me {
     id
     cards(first: 100) {
-      edges { node { ...Card } }
+      edges {
+        node {
+          ...Card
+        }
+      }
     }
   }
 }
 
-mutation CreateCard($source: String!, $address: AddressAttributes) {
-  createCard(source: $source, address: $address) { id }
+fragment SetupIntent on SetupIntent {
+  id
+  status
+  clientSecret
+  nextAction {
+    ...NextAction
+  }
+  paymentMethodTypes
+}
+
+mutation SetupIntent($address: AddressAttributes!) {
+  setupIntent(address: $address) {
+    ...SetupIntent
+  }
+}
+
+fragment PaymentMethod on PaymentMethod {
+  id
+  type
+  isDefault
+  card {
+    ...Card
+  }
+}
+
+query PaymentMethods {
+  account {
```

Comment:
we should ideally have just one account query for all needed data on the billing page (ie paymentMethods, subscription, etc).  You might also need to make it resistent to errors, since i think currently only those w/ billing access can query cards on file